### ISSUE TITLE
Support document.implementation.createHTMLDocument to unlock jQuery usage

### ISSUE THREAD

**Issue by alabiaga** [ampproject/worker-dom#679]
jQuery uses dom.implementation.createHTMLDocument for its parseHtml call which errors when loading the library. Supporting the implementation obj/interface and a textNode on the Node object should unlock jQuery.

**Comment by alabiaga** [comment]
cc\ @kristoferbaxter I'm looking to do the work for this unless there is a reason why this isn't supported. Pls assign to me. Thanks

**Comment by morsssss** [comment]
@alabiaga if this would make jQuery work, um, that would be huge. Any chance you can still work on this?

**Comment by alabiaga** [comment]
@morsssss I started it but just never prioritized it. I'll see how much more work is left and will update this ticket by Monday. Thanks

**Comment by jezeke** [comment]
Has there been any progress on this issue? Looking forward to this being supported.

**Comment by kristoferbaxter** [comment]
@alabiaga Do you have an update?

**Comment by alabiaga** [comment]
Will pass this to @sebastianbenz. I'll sync with him this coming week. Thanks

**Comment by morsssss** [comment]
I'm also just guessing this wouldn't be sufficient to get jQuery to start
working. Wouldn't it be nice though?

**Comment by alabiaga** [comment]
@morsssss  correct. Last time I checked it should at least get it to point of loading the library.
